roles/openshift_facts/library/openshift_facts.py

```diff
@@ -1163,6 +1167,49 @@ def init_local_facts(self, facts=None):
         self.changed = changed
         return new_local_facts
 
+    def init_in_docker_facts(self, facts):
```

Comment:
Everything here should be set only if it isn't already set, otherwise we do not give the user a way to override the settings we are giving them.